code/parachain/frame/assets-registry/src/lib.rs

```diff
@@ -258,12 +261,12 @@ pub mod pallet {
 		fn set_reserve_location(
 			asset_id: Self::AssetId,
 			location: Self::AssetNativeLocation,
-			ratio: Option<Rational>,
+			ratio: Rational,
 			decimals: Option<Exponent>,
 		) -> DispatchResult {
 			ForeignToLocal::<T>::insert(&location, asset_id);
 			LocalToForeign::<T>::insert(asset_id, ForeignMetadata { decimals, location });
-			AssetRatio::<T>::mutate_exists(asset_id, |x| *x = ratio);
+			AssetRatio::<T>::mutate_exists(asset_id, |x| *x = Some(ratio));
```

Comment:
Given that `ratio` is no longer an `Option` shouldn't we also change it in the storage?

Reply:
Resolving as it's been decided not to modify storage for now